polyfill/test/duration.mjs

```diff
@@ -1545,14 +1548,10 @@ describe('Duration', () => {
         equal(oneDay.negated().total({ unit: 'hours', relativeTo }), -25);
       });
       it('start inside repeated hour, end in skipped hour', () => {
-        // gain one hour at the beginning, lose one at the end, days average 24h
-        equal(Duration.from({ days: 126, hours: 1 }).total({ unit: 'days', relativeTo: inRepeatedHour }), 126 + 1 / 24);
-        equal(
-          Duration.from({ days: 126, hours: 1 }).total({ unit: 'hours', relativeTo: inRepeatedHour }),
-          126 * 24 + 1
-        );
+        equal(Duration.from({ days: 126, hours: 1 }).total({ unit: 'days', relativeTo: inRepeatedHour }), 126 + 1 / 23);
```

Comment:
Is `equal` correct here? Or do you want to compare the difference to `Number.EPSILON`?

Reply:
You're right, but I've been procrastinating on doing that because it makes the error message much harder to follow. I should go and do that on all of these when this bug is fixed.